libcontainer/specconv/example.go

```diff
@@ -116,6 +116,7 @@ func Example() *specs.Spec {
 				"/proc/timer_stats",
 				"/proc/sched_debug",
 				"/sys/firmware",
```

Comment:
Paths should be sorted.

Reply:
Since the existing list is not sorted, I wonder sorting in this PR might have some negative impact on git commit traceability.

Reply:
Hmm, fair enough.